logo/README.md

```diff
@@ -18,3 +18,7 @@ It has been built with open source:
 * Something that incorporates the image of water (aqua) and package
 * Minimalistic: should express the idea with less design components
 * Scalable: should look good irrespective of the scale (favicon, t-shirts, print-outs and everything else)
+
+## License of Logo
+
+<a rel="license" href="http://creativecommons.org/licenses/by-nc-nd/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by-nc-nd/4.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-nc-nd/4.0/">Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License</a>.
```

Comment:
This is pretty close to [the official Kubernetes logo license](https://github.com/kubernetes/kubernetes/blob/master/logo/LICENSE).
I think in the future, we might want to remove the `NonCommercial` (if a company wants to use Aqua) and `NoDerivatives` (if someone wants to use modified Aqua logo for their registry) part but it should be okay to keep it for now.

Reply:
Otherwise, LGTM. Thanks for researching about license!

Reply:
Thank you for your review!